pyscal/scalrecommendation.py

```diff
@@ -25,7 +25,7 @@ class SCALrecommendation(object):
         tag (str): A string that describes the recommendation. Optional.
     """
 
-    def __init__(self, low, base, high, tag=None, h=0.01):
+    def __init__(self, low, base, high, tag=None, h=0.01, fast=False):
```

Comment:
This should not really be necessary, as it should be assumed given  in the `low`, `base` and `high` objects. 

The class must decide what to do if `low.fast != base.fast` for example.

Or,  do we need to support overriding `low.fast`?

Reply:
I guess all should be .fast for fast-mode to be activated, but maybe just issue a warning not an error if low.fast != base.fast.